duckduckgo/Android
# ADS: Fireproof Websites screen migration

Task/Issue URL: https://app.asana.com/0/0/1203467500325942/f

### Description
Migration of Fireproof Websites screen to new components

### Steps to test this PR

- Install from this branch
- Go to Settings > Fireproof Websites

### UI changes
| Before  | After |
| ------ | ----- |
![Screenshot_20221202-114053_DuckDuckGo](https://user-images.githubusercontent.com/20798495/205285445-88b27ff9-1484-4bf0-9545-e65b64691feb.jpg)|![Screenshot_20221202-104630_DuckDuckGo](https://user-images.githubusercontent.com/20798495/205285481-7c150f58-5735-49b7-90de-d002faac2b15.jpg)|
![Screenshot_20221202-114120_DuckDuckGo](https://user-images.githubusercontent.com/20798495/205285469-b0e3c39d-0def-434f-b036-61b8f812a2e0.jpg)|![Screenshot_20221202-112656_DuckDuckGo](https://user-images.githubusercontent.com/20798495/205285511-bb3525b3-c251-42bf-9b20-ef571f6f1151.jpg)|
